instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./CommentSection.css";
+
+const CommentSection = props => {
```

Comment:
```suggestion
const CommentSection = ({comments}) => {
```
You can destructure like this better instead of having to create a new line